src/reference/asciidoc/sftp.adoc

```diff
@@ -75,6 +75,9 @@ Under the covers, the SFTP Session Factory relies on the https://mina.apache.org
 However, Spring Integration also supports the caching of SFTP sessions.
 See <<sftp-session-caching>> for more information.
 
+NOTE: The `DefaultSftpSessionFactory` can be based on externally configured `SshClient` or its extension.
+For example, an `org.eclipse.jgit.internal.transport.sshd.JGitSshClient` extension from the `org.eclipse.jgit:org.eclipse.jgit.ssh.apache` library may be used to bring (back) support for HTTP/SOCKS proxies.
```

Comment:
```suggestion
For example, the `org.eclipse.jgit.internal.transport.sshd.JGitSshClient` extension from the `org.eclipse.jgit:org.eclipse.jgit.ssh.apache` library may be used to provide support for HTTP/SOCKS proxies.
```

Just to clarify... "...an `...JGitSshClient` from the..." would be correct English but, here, the subject is the extension and the implementation describes it.